data-transfer-object/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: pattern
+title: Data transfer object
+folder: data-transfer-object
+permalink: /patterns/data-transfer-object/
+categories: Creational
+tags: 
+ - Java
+ - Difficulty-Beginner 
+---
+
+## Also known as
+Virtual Constructor
```

Comment:
I doubt whether it is called Virtual constructor. Virtual constructor is a term for Factory pattern. DTOs are also known as Transfer Objects or vaguely as Value Objects. Can we correct this?